### DISCUSSION TITLE
Pagination: Jump to first/last page

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#4279] [Q&A] [answered]
- Laravel Version: 9
- Nova Version: 4.6
- PHP Version: 8

### Description:
Since the move to Nova 4, it's impossible to move to the first/last page of a collection in one click.
I need to click next/previous many time or use the search field.
Please bring back to First/Last page button instead of only Previous/Next.
Or add maybe if we click the `1-25 of 91`, there could be a popup when you can type the page we want to go to.
It would be nice to also bring back the clickable page number.
I'm not sure why those feature were removed.

Nova 4:
![Capture d’écran, le 2022-05-18 à 13 21 55](https://user-images.githubusercontent.com/17392251/169103989-a11e58af-b9a9-4863-b491-a2e313c0f7fb.png)

Nova 3:
![Capture d’écran, le 2022-05-18 à 13 24 03](https://user-images.githubusercontent.com/17392251/169104510-c7d31e2b-65cc-43e5-9889-f4357e27329b.png)

**Comment by crynobone** [comment, marked as answer]
![silver-surfer test-NovaSite-Users](https://user-images.githubusercontent.com/172966/169174534-6a92ab75-7618-4a3b-82a7-9dff10cde350.png)

Please change your pagination config to `links`:

```php
    /*
    |--------------------------------------------------------------------------
    | Nova Pagination Type
    |--------------------------------------------------------------------------
    |
    | This option defines the visual style used in Nova's resource pagination
    | views. You may select between "simple", "load-more", and "links" for
    | your applications. Feel free to adjust this option to your choice.
    |
    */

    'pagination' => 'links',
```

**Reply by mrleblanc101** [reply]
Ok thank you, feels like this should be the default.
I mean jumping to the last/first page seems pretty essential for a collection view.
